src/main/java/com/jalasoft/pivotal/pages/account/AccountsList.java

```diff
@@ -0,0 +1,60 @@
+package com.jalasoft.pivotal.pages.account;
+
+import com.jalasoft.pivotal.pages.AbstractPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AccountsList extends AbstractPage {
+
+    @FindBy(css = "a[id='new_account_link']")
```

Comment:
you can use directly
`#new_account_link`

Reply:
Done